Kajabi/sage-archived
# Align Avatars to Spec

## Description

Aligned the Avatar object to spec

- [x] - align to `xsmall-bold` `font-size`
- [x] - center svg `<text>` within parent
- [x] - remove the avatar ring
- [x] - remove images as only initials are called for
- [x] - create Sage Rails component variation
- ~[ ] - create Sage React component variation~ will address in [PR 16098](https://github.com/Kajabi/kajabi-products/pull/16098)
- [x] - update documentation, examples, and properties

### Screenshots


|  before  |  after  |
|--------|--------|
|![Screen Shot 2020-10-14 at 4 01 18 PM](https://user-images.githubusercontent.com/1241836/96044748-8ce87700-0e36-11eb-92f8-8c97df1555c3.png)|![Screen Shot 2020-10-14 at 4 00 22 PM](https://user-images.githubusercontent.com/1241836/96044765-92de5800-0e36-11eb-87a8-3f8c46d82a40.png)|


## Test notes


## Spec
- [Avatar Spec ](https://www.figma.com/file/SCZt9jMd7OcTMrNsOFifDp/Sage-Components-WIP?node-id=861%3A5442)

### Steps for testing
1. Visit the Objects -> Avatar page: http://localhost:4000/pages/object/avatar
2. Observe that the Avatars align with the spec
